apps/tailwind-components/components/input/listbox/index.vue

```diff
@@ -0,0 +1,369 @@
+<template>
```

Comment:
So this component will be 'InputListbox'?

I would be happier if we  move it to /components/input/Listbox